source/Halibut.Tests.DotMemory/MemoryFixture.cs

```diff
@@ -20,16 +20,22 @@ public interface ICalculatorService
     {
         long Add(long a, long b);
     }
+    
+    public interface IAsyncCalculatorService
+    {
+        Task<long> AddAsync(long a, long b, CancellationToken cancellationToken);
+    }
 
-    public interface IAsyncClientCalculatorService
+    public interface IClientAsyncClientCalculatorService
```

Comment:
@nathanwoctopusdeploy I don't see why this interface got renamed. All other interfaces are `IAsyncClient..... Service` style. Let's put it back?

Reply:
Agreed

Reply:
Done